test-app/tests/integration/components/x-toggle/component-test.js

```diff
@@ -244,86 +242,85 @@ module('Integration | Component | x toggle', function (hooks) {
         </XToggle>
       `);
 
-      await click('.on-label');
-      assert.true(this.value, 'clicking on label toggles value true');
-      assert.strictEqual(this.timesCalled, 1, 'should call onToggle once');
-
-      await click('.on-label');
-      assert.true(this.value, 'clicking on label again stays true');
-      assert.strictEqual(
-        this.timesCalled,
-        1,
-        'should not call onToggle again if value does not change'
-      );
-    });
+    await click('.on-label');
+    assert.true(this.value, 'clicking on label toggles value true');
+    assert.strictEqual(this.timesCalled, 1, 'should call onToggle once');
+
+    await click('.on-label');
+    assert.true(this.value, 'clicking on label again stays true');
+    assert.strictEqual(
+      this.timesCalled,
+      1,
+      'should not call onToggle again if value does not change'
+    );
+  });
 
-    test('swipe gestures', async function (assert) {
-      this.set('value', false);
+  test('swipe gestures', async function (assert) {
```

Comment:
@nickschot seem to be having issues with the gestures addon? No error but seems to not pass the tests any more. See test run here https://github.com/knownasilya/ember-toggle/actions/runs/3494201805/jobs/5849786261#step:6:29

Reply:
I think it has to do with styles. I forgot to add them to the new reorg 🙈